### PR TITLE
Feature/mongoid7

### PR DESCRIPTION
**Pull request by wuhuizuo**
adapt to mongoid `v7.0`

### CODE REVIEW

**Comment by wuhuizuo** [comment]
the ci failed, but not on the unchanged old mongoid branches

File: .travis.yml [hunk 1]
```diff
@@ -23,6 +23,10 @@ matrix:
       env: MONGOID_VERSION=6
       before_script:
         - bundle exec danger
+    - rvm: 2.3.1
+      env: MONGOID_VERSION=7
+      before_script:
```
Comment (dblock) [inline .travis.yml]: We only need to run `danger` once in the whole build.

Comment (wuhuizuo) [reply .travis.yml]: OK, I will fix it. but ruby versions also need  to consider v2.4.x and v2.5.x. Thhere are some diffrences between 2.4 and 2.5.

**Comment by wuhuizuo** [comment]
@CI PLS REBUILD

**Comment by dblock** [comment]
There're a few references to versions of mongoid in README. Also please update https://github.com/mongoid/mongoid-locker/blob/master/CHANGELOG.md.

**Comment by wuhuizuo** [comment]
Documents updated:
- README.md
- CHANGELOG.md
- LICENSE.txt (year part)

**Comment by wuhuizuo** [comment]
@dblock PLS re checking it.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ### 0.3.6 (Next)
 
-* Your contribution here.
```
Comment (dblock) [inline CHANGELOG.md]: Can you please put this one back ;)

**Comment by dblock** [comment]
Looks good, just that change thing pls.

**Comment by dblock** [comment]
Merged, thanks.

**Comment by wuhuizuo** [comment]
@dblock When release a new version?

**Comment by dblock** [comment]
Opened https://github.com/mongoid/mongoid-locker/issues/54, any maintainer can do, I might get to it soon

**Pull request merged**